jest-puppeteer.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   launch: {
     headless: true,
     product: 'chrome',
-  }
+    devtools: false
```

Comment:
launch without devtools will be faster?

Reply:
devtool cannot set to true when headless is true